manifests/vhost.pp

```diff
@@ -1794,7 +1794,7 @@
   Optional[Integer] $limitreqbody                                                     = undef,
   Optional[String] $proxy_dest                                                        = undef,
   Optional[String] $proxy_dest_match                                                  = undef,
-  Optional[String] $proxy_dest_reverse_match                                          = undef,
+  Optional[Variant[Array[Hash], Hash]] $proxy_dest_reverse_match                      = undef,
```

Comment:
Hey, this looks good to me but could you add String back as a variant? Otherwise it's backwards incompatible.
You can not this in the below issue and we will correct it properly upon the next major release:
https://github.com/puppetlabs/puppetlabs-apache/issues/2291